### DISCUSSION TITLE
Click to go the file，Nothing happens after clicking?

### DISCUSSION THREAD

**Discussion by warmthsea** [vuejs/devtools#198] [General]
![image](https://github.com/vuejs/devtools-next/assets/45450994/3b36d015-345e-444a-bcbd-22d98d1dfee7)

**Comment by webfansplz** [comment]
See: https://devtools-next.vuejs.org/guide/FAQ#i-can-t-use-the-open-in-editor-feature

**Reply by warmthsea** [reply]
Thanks Reply.
